server/compactor/src/main/java/org/apache/accumulo/compactor/Compactor.java

```diff
@@ -386,6 +386,8 @@ protected void updateCompactionState(TExternalCompactionJob job, TCompactionStat
             ThriftUtil.returnClient(coordinatorClient, getContext());
           }
         });
+    LOG.trace("Attempting to update compaction state in coordinator ECID:{}",
```

Comment:
ExternalCompactionId has a prefix of `ECID:` already. I think that gets returned in the String from the `toString` method. If so, then this would end up with two `ECID` in the log statement.

Reply:
Confirmed behavior and removed the duplicate prefix.  fixed in e31aeed85c